internal/graderd/service_handlers.go

```diff
@@ -0,0 +1,51 @@
+package graderd
+
+import (
+	"context"
+	"github.com/Capstone-auto-grader/grader-api-v2/internal/grader-task"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	pb "github.com/Capstone-auto-grader/grader-api-v2/graderpb"
+)
+
+// SubmitForGrading fulfills a SubmitForGradingRequest.
+func (s *Service) SubmitForGrading(ctx context.Context, req *pb.SubmitForGradingRequest) (*pb.SubmitForGradingResponse, error) {
+	if err := req.Validate(); err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	// Create tasks.
+	taskList := make([]grader_task.Task, 0, len(req.GetTasks()))
+	for _, t := range req.GetTasks() {
+		task := grader_task.NewTask(t.GetAssignmentId(), t.GetZipKey(), t.GetTestKey(), t.GetStudentName(), t.GetCallbackUri(), t.GetTimeout())
+		taskList = append(taskList, task)
+		_ = s.schd.StartTask(ctx, task.ID)
```

Comment:
im not sure if we are ignoring the error here because 1) we don't want the error in one task to fail other tasks, or 2) we just don't know how to handle a start task error yet.
If we decide that it is 1), we need to figure out a way to let the caller know that they have to retry for _that_ specific task unless we retry internally.